db/migrate/20220212200515_create_posts.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
+    end
```

Comment:
- [ ] The post table must have the following fields: `title`, `text`,  `comments_counter`, and `likes_counter`. Kindly update the migration to accommodate these fields as per the requirements.
- [ ] User reference should resemble the code below
```ruby
t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index (but students can also add column and index in separate commands)`
```